pyproject.toml

```
@@ -6,7 +6,7 @@ version = "1.1.0"
 license = {text = "MIT"}
 requires-python = ">=3.9"
 dependencies = [
-  "toml>=0.7.0,<3.11.0",
+  "tomli>=2.2.0,<3.11.0",
```

tomli need only for python 3.11
```suggestion
  "tomli>=2.2.0; python_version < "3.11",
```